upspeak/index.html

```diff
@@ -44,7 +44,7 @@
 <section class="section pb-10 bg-gradient-white">
     <div class="container-fluid">
         <div class="row row-grid align-items-center justify-content-center">
-            <div class="col-md-4">
+            <div class="col-lg-5">
```

Comment:
We need to keep `col-md-4` as well.
```suggestion
            <div class="col-lg-5 col-md-4">
```